docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+services:
+    django:
+        build: ./edagames-django
+        ports:
+            - "8000:8000"
+        volumes: 
+            - ${PWD}/edagames-django/db.sqlite3:/edagames-django/db.sqlite3
```

Comment:
${PWD} could be replaced by ./

Reply:
We discovered that to mount files with Docker, Docker needs absolute paths.
 **./** works as a relative path
 **${PWD}:** works as absolute path